packages/teuchos/comm/src/Teuchos_TimeMonitor.cpp

```diff
@@ -212,15 +212,17 @@ namespace Teuchos {
     }
   };
 
+  constexpr const std::string_view defaultStackedTimerName = "Teuchos::StackedTimer";
```

Comment:
Not being in an anonymous namespace, is it possible to create a name collision in the binaries?  Seems very unlikely but this is really just an implementation detail and therefore should be in an anonymous namespace if you are being pedantic.  But since the risk is low given this is in the Teuchos namespace, this is not a big deal.